src/function/algebra/resolve.js

```diff
@@ -35,11 +35,12 @@ export const createResolve = /* #__PURE__ */ factory(name, dependencies, ({
    *
    *     simplify, evaluate
    *
-   * @param {Node} node
-   *     The expression tree to be simplified
+   * @template nodeOrNodes
+   * @param {nodeOrNodes} node
+   *     The expression tree (or trees) to be simplified
```

Comment:
I'd suggest keeping this less "fancy" and just writing @param { Node | Node[] } nodeOrNodes, but I am not picky either way as long as the web doc generation works -- please confirm that it does, thanks.